frontend/src/components/CollectionRow.jsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { connect } from "react-redux";
+import { actions } from "../store";
+
+const CollectionRow = ({ collectionId, handleDeleteCollection }) => {
+  const creator = creators.find(creator.id === collectionId);
+
+  return (
+    <tr>
+      <td>creators.fullName</td>
+      <td>[Description]</td>
```

Comment:
just a note for future reference, description will come from creatorInsights (not creators), which is currently included in creators.

Reply:
so are we using 'creator.creatorInsights.biography ' as a description?

Reply:
yup!